src/map/status.cpp

```diff
@@ -6285,6 +6517,8 @@ static signed short status_calc_critical(struct block_list *bl, struct status_ch
 #ifdef RENEWAL
 	if (sc->data[SC_SPEARQUICKEN])
 		critical += 3*sc->data[SC_SPEARQUICKEN]->val1*10;
+	if (sc->data[SC_TWOHANDQUICKEN])
+		critical += sc->data[SC_TWOHANDQUICKEN]->val1 + 1;
```

Comment:
this should be 
		critical += sc->data[SC_TWOHANDQUICKEN]->val1 + 1;
To increase the rate by lvl up to 10 crit al lvl 10